src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -62,7 +62,10 @@ public static class TextCatalog
             TextFeaturizingEstimator.Options options,
             params string[] inputColumnNames)
             => new TextFeaturizingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(),
-                outputColumnName, inputColumnNames, options);
+                outputColumnName,
+                inputColumnNames == null || inputColumnNames.Length == 0 || (inputColumnNames.Length == 1 && inputColumnNames[0] == "")  ?
```

Comment:
>inputColumnNames == null || inputColumnNames.Length == 0 [](start = 16, length = 56)

These two conditions can be replaced by `Utils.Size(inputColumnNames) == 0`.

Reply:
Thank you for the feedback, I will change that in my next update.